ui/v2/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -65,13 +68,36 @@ export const SettingsConfigurationPanel: FunctionComponent<IProps> = (props: IPr
       setLogOut(conf.general.logOut);
       setLogLevel(conf.general.logLevel);
       setLogAccess(conf.general.logAccess);
+      setExcludes(conf.general.excludes);
     }
   }, [data]);
 
   function onStashesChanged(directories: string[]) {
     setStashes(directories);
   }
 
+  function excludeRegexChanged(idx: number,value: string) {
+	const newExcludes = excludes.map((regex,i)=> {  
```

Comment:
We use spaces for tabs with a tab-size of 2 for the UI files. Tabs appear throughout the new code. Also for consistent formatting, add a space after `,` characters.

Reply:
Noted.
Should be ok now.